### PR TITLE
Update setup

### PR DESCRIPTION
**Pull request by hcastilho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,113 +43,160 @@ If you are using MacOS or Linux you can skip this section.
...
 ![Create Repository](assets/create_repository.png "Create Repository")
 
 
 #### Cloning the Workspace
 
-1. Open a Terminal or Git Bash, the next steps are on this terminal
-1. Clone your _<username>/batch3-workspace_ repository
+1. Open a Terminal
+1. Chose where you want to keep your project, if for example you want to keep
+it in a directory called _Projects_ you would do the following.
```
[the remark this answers is not in the document]
Comment (hcastilho) [reply README.md]: Agreed

File: README.md [hunk 2]
```diff
@@ -38,113 +43,160 @@ If you are using MacOS or Linux you can skip this section.
...
+In the `ds-prep-course` repository that you just cloned there is a sample
 learning unit.
 It's used to give instructors guidelines to produce the learning units.
 We are also using it to ensure that you are able to run and submit a learning 
 unit.
 
 So go ahead and copy the sample directory `sample/SLU00 - LU Tutorial` from the `
-batch3-students` repository to your repository (named `batch3-workspace`).
+ds-prep` repository to your repository (named `ds-prep-workspace`).
 ![Sample learning unit](assets/sample_learning_unit.png "Sample learning unit")
```
[the remark this answers is not in the document]
Comment (hcastilho) [reply README.md]: Typo in the image name

File: README.md [hunk 3]
```diff
@@ -38,113 +43,160 @@ If you are using MacOS or Linux you can skip this section.
...
 ### Running Learning Unit
 
-In the `batch3-students` repository that you just cloned there is a sample
+In the `ds-prep-course` repository that you just cloned there is a sample
 learning unit.
 It's used to give instructors guidelines to produce the learning units.
 We are also using it to ensure that you are able to run and submit a learning 
 unit.
 
 So go ahead and copy the sample directory `sample/SLU00 - LU Tutorial` from the `
```
[the remark this answers is not in the document]
Comment (hcastilho) [reply README.md]: sure thing, but it would have to be severall commnads or one very exotic

**Comment by buedaswag** [comment]
looks good!

**Pull request merged**